Added alt text, additional information to step 2, and button to step 3 (#992)
CHTC/chtc-website-source: _uw-research-computing/htc-roadmap.md

---
highlighter: none
layout: guide
title: Roadmap to getting started
guide:
  category: Get started
  tag:
    - htc
---

### Step One: Introduction to the High Throughput Computing Strategy

Like nearly all large-scale compute systems, users of both CHTC's High Throughput and High Performance systems prepare their computational work and submit them as tasks called jobs. These jobs run on execution points, which are the computers that perform the work.

<p style="text-align:center"><img src="/images/roadmap-workflow-overview.png" alt="The process of submitting a job including the access point, job queue, and execution points." width=900px></p>

**Terminology:**

> - **Access point** is where you login and stage your data, executables/scripts, and software to use in jobs. 

Reviewer: ```suggestion
- The **Access Point** is where you log in and stage your data, executables/scripts, and software to use in jobs. 
```

- **HTCondor** is a job scheduling software that will run your jobs out on the execution points. 

> - The **Execution Points** is the set of resources your job runs on. It is composed of servers, as well as other technologies, that compose the cpus, memory, and disk space that will run the computations of your jobs. 

Reviewer: ```suggestion
- The **Execution Points** are the set of resources your job runs on.
```
I think the latter half of the definition for EPs might be a bit too much information for a brand new user. Let's omit for now.

> High Throughput Computing systems specialize in running several small, independent jobs. On the other hand, High Performance Computing systems specialize in running a few, very large jobs that use multiple computers working together.

Reviewer: ```suggestion
High Throughput Computing systems specialize in running tens to millions small, independent jobs. On the other hand, High Performance Computing systems specialize in running a few, very large jobs that use multiple computers working together on the same problem.
```

<table class="comparison-table">
   <thead>
      <tr>
      <th></th>
      <th>HTC (High Throughput Computing)</th>
      <th>HPC (High Performance Computing)</th>
      </tr>
   </thead>
   
   <tbody>
      <tr>
      <td>Best for...</td>
      <td>Running several independent jobs</td>
      <td>Running one very large job</td>
      </tr>
      <tr>
      <td>Typical size</td>
      <td>1 - 4 CPUs/job</td>
      <td>30+ CPUs/job</td>
      </tr>

>       <td>Computers work together?</td>

Reviewer: ```suggestion
      <td>Multiple computers work together on one task?</td>
```

      <td>X</td>
      <td>✓</td>
      <tr>
      <td>Examples</td>
      <td>Simulations, image processing, and machine learning workflows</td>
      <td>Climate modeling, fluid dynamics, and large optimizations</td>
      </tr>
   </tbody>
</table>

> It is important to keep this distinction in mind when setting up your jobs. On the HTC system, smaller jobs that request fewer resources (CPU, memory, and disk) are generally easier to find a slot to run on.

Reviewer: ```suggestion
It is important to keep this distinction in mind when setting up your jobs. On the HTC system, smaller jobs that request fewer resources (CPU, memory, and disk) are generally easier to find a slot to run on, so they start more quickly, and you can have many of them running at the same time.
```
